aclocal.m4

```diff
@@ -1,6 +1,6 @@
-# generated automatically by aclocal 1.16.3 -*- Autoconf -*-
+# generated automatically by aclocal 1.16.1 -*- Autoconf -*-
```

Comment:
@corona10, what distribution and version of Linux or BSD did you use [to upgrade aclocal version five month ago](https://github.com/python/cpython/commit/b2cf2513f9184c850a69fab718532b4f7c6a003d#diff-d05beee91e3fb3e91d08b841553b74a24b9cfb9f3d238314e53229f2cf416475L1-R1)? This will be useful for @8vasu to install the exact distro into a virtual machine.

@8vasu For some reason, a minor 1.16.3→1.16.1 version rollback causes a big unnecessary diff out of the blue. It would be great if you could regenerate the file once more but with aclocal 1.16.3 since we already sticked to more recent version.

Reply:
However... @tiran, what version of aclocal is better to stick to? According to `git blame`, the version jumped like a rabbit: 1.16.3 ← 1.16.1 ← 1.16.2 ← 1.15 ← 1.16.1 ← 1.15 ← 1.15.1 ← 1.15 ← 1.15.1 ← 1.15 ← 1.15.1 ← 1.15 ← 1.14.1.

Reply:
Soumendra has an older version of autoconf and pkgconf's m4 package is missing. You can use my container image https://quay.io/repository/tiran/cpython_autoconf to regenerate autoconf files.

Reply:
We did not enforce a specific version in the past. Nowadays we have one semi-official container image that we use to regenerate the files.

Reply:
I will get this done in a few hours after I get back home.

Reply:
Done. Also, the `docker run` command mentioned in https://quay.io/repository/tiran/cpython_autoconf kept generating errors; I think there is no `--pull` option for `docker run`; however, there seems to be one for `podman run`. I referred to https://docs.docker.com/engine/reference/run/ and https://docs.podman.io/en/latest/markdown/podman-run.1.html. I do not have `podman`, but the docker command worked fine without the `--pull` option.